File: back/src/dtos/roster_dto.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
#[derive(Debug)]
pub struct RosterDTO {
    pub roster_id : Option<i32>,
    pub campus_id : i32,
    pub teacher_name : String,
    pub teacher_lastname1 : String,
    pub teacher_lastname2 : String,
    pub subject_name : String,
    pub uni_structure_id : i32,
    pub structure_name : String
}

File: back-csharp/_dtos/RosterDto.cs
namespace back_csharp._dtos;

public class RosterDto
{ 
        public int RosterId { get; set; }
        public Guid RecordId { get; set; }
        
        public string TeacherName { get; set; }

        public string TeacherLastname1 { get; set; }

        public string TeacherLastname2 { get; set; }


        public int CampusId { get; set; }
        
        public string CampusName { get; set; }

}